recipe/meta.yaml

```diff
@@ -31,6 +32,7 @@ requirements:
     - tk
   run:
     - python
+    - vs2015_runtime 14.00.23026.0 0   # [win and py35]
```

Comment:
I am not against you merging this as an emergency measure. I just want to be clear on what should be done in the long term.